src/appendix/humorust.md

```diff
@@ -10,4 +10,4 @@ enlightening?
 - [Rust Koans](https://users.rust-lang.org/t/rust-koans/2408)
 - [`break rust;`](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=0ab2bd6a9d722e0f05a95e2a5dcf89cc)
 - [The Nomicon Intro](https://doc.rust-lang.org/stable/nomicon/)
-- [`rustc-ty` renaming punfest](https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/rustc-ty.20naming.20bikeshed.20.2F.20punfest.20\(was.3A.20design.20meeting.202.2E.2E.2E/near/189906455)
+- [`rustc-ty` renaming punfest](https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/rustc-ty.20naming.20bikeshed.20.2F.20punfest.20%28was.3A.20design.20meeting.202.2E.2E.2E/near/189906455)
```

Comment:
This is still broken: https://rustc-dev-guide.rust-lang.org/appendix/humorust.html
It works fine with encoded `(` on my local.